parlai/zoo/blender/dict_3B.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+"""
+3B parameter Blender dictionary file: please see <parl.ai/project/blender>.
```

Comment:
hmm, I've never seen this URL notation before - https://parl.ai/project/blender gives me a "Page not found"